migrations/m171028_191358_add_avatar_for_user.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+use yii\db\Migration;
+
+class m171028_191358_add_avatar_for_user extends Migration
+{
+    public function safeUp()
+    {
+        $this->addColumn('{{%user}}', 'avatar', $this->string(60)->after('fullname'));
```

Comment:
Why 60?

Reply:
update it to 16;
max int value for user_id plus file extension => '123123.jpg'

16 chars will be enough